functions/vpn.bash

```diff
@@ -169,3 +169,61 @@ setup_wireguard() {
   echo -n "$(timestamp) [openHABian] Generating QR to load config on the client side (download Wireguard app from PlayStore or AppStore)... "
   qrencode -t ansiutf8 </etc/wireguard/wg0-client.conf
 }
+
+
+## Install Tailscale from their own repo
+## Valid arguments: "install" or "remove"
+##
+##   install_tailscale(String action)
+##
+install_tailscale() {
+  local queryText="We will install the tailscale VPN client on your system. Use it to securely interconnect multiple openHAB(ian) instances.\\nSee https://tailscale.com/blog/how-tailscale-works/ for a comprehensive explanation how it creates a secure VPN. For personal use, you can get a free solo service from tailscale.com."
```

Comment:
Unused

Reply:
what is unused ?

Reply:
queryText, right?

Reply:
No I think it was in use. Anyway, I just changed that.